src/info/repo/author.rs

```diff
@@ -135,4 +135,80 @@ mod test {
 
         assert_eq!(author.to_string(), "75% John Doe 1500");
     }
+
+    #[test]
+    fn test_author_info_title() {
+        let author = Author::new(
+            "John Doe".into(),
+            "john.doe@email.com".into(),
+            1500,
+            2000,
+            true,
+        );
+
+        let author2 = Author::new(
+            "Roberto Berto".into(),
+            "bertolone2000@email.com".into(),
+            240,
+            300,
+            false,
+        );
+
+        let authors_info = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author.clone()],
+        };
+
+        let authors_info_2 = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author2, author],
+        };
+
+        assert_eq!(authors_info.title(), "Author");
+        assert_eq!(authors_info_2.title(), "Authors");
+    }
+
+    #[test]
+    fn test_author_info_value() {
+        let author = Author::new(
+            "John Doe".into(),
+            "john.doe@email.com".into(),
+            1500,
+            2000,
+            true,
+        );
+
+        let author2 = Author::new(
+            "Roberto Berto".into(),
+            "bertolone2000@email.com".into(),
+            240,
+            300,
+            false,
+        );
+
+        let authors_info = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author.clone()],
+        };
+        let authors_info_2 = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author, author2],
+        };
+
+        println!("{}", authors_info);
+        println!("{}", authors_info_2);
+
+        assert_eq!(
+            authors_info.value(),
+            "\u{1b}[38;2;255;0;0m75% John Doe <john.doe@email.com> 1500\u{1b}[39m"
+        );
+
+        let pad = authors_info_2.title().len() + 2;
+        let mut pad_str = String::from("");
+        let perc = (240 as f32 * 100. / 300 as f32).round() as usize;
+        for i in 0..pad {
+            pad_str.push_str(" ");
+        }
+        assert_eq!(authors_info_2.value(), format!("\u{1b}[38;2;255;0;0m75% John Doe <john.doe@email.com> 1500\u{1b}[39m\n{}\u{1b}[38;2;255;0;0m{}% Roberto Berto 240\u{1b}[39m", pad_str, perc))
```

Comment:
Also use `.contains` here.

This test would be more readable if you just hardcoded the expected percentage, too. Because you control the factors (author's name, etc.) it's OK, even preferable, to hardcode the expected values.

So just make `perc` a string and test that `{perc}%` is contained.